deps/build.jl

```diff
@@ -1,16 +1,21 @@
 using Compat
 
+const condadir = abspath(homedir(), ".julia", "conda")
```

Comment:
Why not `joinpath(DEPOT_PATH[1], "conda")`? But maybe it's fine since `julia` itself hard-codes `~/.julia/config/startup.jl` https://github.com/JuliaLang/julia/blob/b519147d60683c3a0ed587925bb7221153b64b65/base/client.jl#L288